sdk/file_sdk/OPCFileFormat/OPCFileFormat.pro

```diff
@@ -5,6 +5,7 @@ TARGET    = modopcfile
 TEMPLATE = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
 INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI $$REPO_ROOT/sdk/file_sdk $$REPO_ROOT/sdk/file_sdk/Common
+INCLUDEPATH += $$REPO_ROOT/sdk/rms_sdk/Profile
```

Comment:
nit: same as the others

Reply:
Yup. Its not worth it. This is just temporary